modules/virt-resources-monitored-for-top-consumption.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assembly:
+//
+// * virt/logging_events_monitoring/virt-reviewing-virtualization-overview.adoc
+
+[id="virt-resources-monitored-for_top_consumption_{context}"]
+= Resources monitored for top consumption
+
+The *Top Consumers* cards in *Virtualization Overview* page display projects, virtual machines or nodes with maximum consumption of a resource. You can select a project, a virtual machine, or a node and view the top five or top ten consumers of a specific resource.
+
+[NOTE]
+====
+Viewing the maximum resource consumption is limited to the top five or top ten consumers within each *Top Consumers* card.
+====
+
+The following table shows resources monitored for top consumers.
+
```

Comment:
I just noticed that a lot of this information also appears in [About reviewing top consumers](https://docs.openshift.com/container-platform/4.10/virt/logging_events_monitoring/virt-reviewing-vm-dashboard.html#virt-about-reviewing-top-consumers_virt-reviewing-vm-dashboard). Has the OpenShift Virtualization dashboard been replaced by the Virtualization Overview page? If so, we might have to deprecate the dashboard.